### PR TITLE
ci: Improve PR description for automated PRs

### PR DESCRIPTION
**Pull request by mqasimsarfraz**
Improve the PR description for automated PRs.

### CODE REVIEW

File: .github/workflows/update-helm-index.yml [hunk 1]
```diff
@@ -46,7 +46,14 @@ jobs:
...
 
-            We released a new version of Inspektor Gadget, so the helm repository index needs to be updated.
+            We released a new version of Inspektor Gadget [${{ steps.bump_inspektor_gadget.outputs.version }}](https://github.com/inspektor-gadget/inspektor-gadget/releases/tag/${{ steps.bump_inspektor_gadget.outputs.version }}), so helm index needs to be updated.
+
+            This PR was automatically generated by a CI cron job:
+            * [update-helm-index.yml](https://github.com/inspektor-gadget/charts/blob/main/.github/workflows/update-helm-index.yml)
+
+            This PR will not be automatically merged. Please review, edit and merge as appropriate.
+            After merging, the following will be updated:
+            * [GitHub Page for Charts](https://github.com/inspektor-gadget/charts/actions/workflows/pages/pages-build-deployment)
```
Comment (alban) [inline .github/workflows/update-helm-index.yml:56]: Did you mean to use this link instead?
```suggestion
            * [GitHub Page for Charts](https://github.com/inspektor-gadget/charts)
```

Comment (mqasimsarfraz) [reply .github/workflows/update-helm-index.yml]: Right. Done, thanks!

File: .github/workflows/update-helm-index.yml [hunk 2]
```diff
@@ -46,7 +46,14 @@ jobs:
...
 
-            We released a new version of Inspektor Gadget, so the helm repository index needs to be updated.
+            We released a new version of Inspektor Gadget [${{ steps.bump_inspektor_gadget.outputs.version }}](https://github.com/inspektor-gadget/inspektor-gadget/releases/tag/${{ steps.bump_inspektor_gadget.outputs.version }}), so helm index needs to be updated.
+
+            This PR was automatically generated by a CI cron job:
+            * [update-helm-index.yml](https://github.com/inspektor-gadget/charts/blob/main/.github/workflows/update-helm-index.yml)
+
+            This PR will not be automatically merged. Please review, edit and merge as appropriate.
+            After merging, the following will be updated:
+            * [GitHub Page for Charts](https://inspektor-gadget.github.io/charts/index.yaml)
```
Comment (alban) [inline .github/workflows/update-helm-index.yml:56]: Do you mean html instead of yaml?

Comment (mqasimsarfraz) [reply .github/workflows/update-helm-index.yml:56]: Yes, This PR updates the `index.yaml` so one can verify once the changes are merged. What do you think?

Comment (alban) [reply .github/workflows/update-helm-index.yml:56]: Ah ok, that's fine then.
I merely wanted to have a way to discover the domain part of the URL.

**Pull request merged**